crates/turborepo-lib/src/cli/error.rs

```diff
@@ -28,11 +28,21 @@ pub enum Error {
         config_path: AbsoluteSystemPathBuf,
         error: io::Error,
     },
+    #[error("Encountered an IO error while attempting to read {auth_path}: {error}")]
+    FailedToReadAuth {
+        auth_path: AbsoluteSystemPathBuf,
+        error: io::Error,
+    },
```

Comment:
Just confirming that we expect `FailedToReadConfig` and `FailedToReadAuth` to diverge in the immediate future. If not, then I don't think we should copy-pasta the error variant.

Reply:
I think so. I'm setting this up now for that potential in the immediate future, and if at the end of this little adventure we find that they don't really diverge then I'm open to getting rid of it, but I think some of this stuff will be moved around to the auth crate anyway.